src/modules/aws_ec2/mappers/instance.ts

```diff
@@ -91,6 +102,35 @@ export class InstanceMapper extends MapperBase<Instance> {
       ));
     out.hibernationEnabled = instance.HibernationOptions?.Configured ?? false;
     out.region = region;
+
+    // check if we can find the instance in database
+    if (instance.InstanceId) {
+      const instanceObj = await this.module.instance.db.read(
```

Comment:
this could be 
```suggestion
      const instanceObj = await this.db.read(
```
right?